app/Http/Controllers/HomeController.php

```diff
@@ -236,7 +236,7 @@ public function supportTheGame()
                 'perks' => [
                     // localization's name => icon
                     'osu_direct' => 'fas fa-search',
-                    'auto_downloads' => 'fas fa-cloud-download-alt',
+                    'auto_downloads' => 'fas fa-download',
                     'upload_more' => 'fas fa-cloud-upload-alt',
```

Comment:
regular upload icon too

Reply:
That page is being replaced "soon" anyway.